rest/config_startup.go

```diff
@@ -135,6 +135,7 @@ type ReplicatorConfig struct {
 }
 
 type UnsupportedConfig struct {
+	Serverless        *bool                `json:"serverless,omitempty" help:"Put SG in to serverless mode."`
```

Comment:
This probably needs to be a field on `DatabaseContextOptions` too, so that it's accessible from a database - and passed through via `dbcOptionsFromConfig`